microsoft-edge/webview2/how-to/webdriver.md

```diff
@@ -93,23 +108,65 @@ By this point, you've installed the WebView2 Runtime, built a WebView2 project,
+
+In this scenario, there's one WebView2 instance, and it is available upon launch without needing to navigate through any native UI.
+
+### Approach 2: Attaching Microsoft Edge Driver to a running WebView2 app
+<!-- preferred phrase -->
+
+If you have any situation that doesn't fit the "launch" scenario above, you should attach Microsoft Edge Driver to a running WebView2 instance (instead of letting Microsoft Edge Driver handle the WebView2 launch); use [Step 4b: Attaching Microsoft Edge Driver to a running WebView2 app](#step-4b-attaching-microsoft-edge-driver-to-a-running-webview2-app).
+
+Some examples of scenarios that don't fit the "launch" scenario are:
+*  You need to navigate through some native UI before the WebView2 instance is created.
+*  Your app creates multiple WebView2 instances, and you want to attach to a specific instance.
+
+In such scenarios, we recommend attaching to a specific instance of WebView2, because having Microsoft Edge Driver launch your WebView2 app is for relatively simple scenarios only.  When Microsoft Edge Driver launches your app, it automatically attaches to the first WebView2 instance that is created, and will fail if no WebView2 instance is found.
+
+Regardless of whether you use the "launch" or "attach" approach, you must know Selenium, download Microsoft Edge Driver, and make sure the versions match.  What you do with your WebDriver framework, such as Selenium, will differ between the "launch" versus "attach" approaches.
+
+
+<!-- ====================================================================== -->
+## Step 4a: Letting Microsoft Edge Driver launch your WebView2 app
+<!-- old title: Drive WebView2 with Selenium and Microsoft Edge Driver -->
+
+Note: Selenium with C# is just one of many WebDriver testing frameworks.
+
+Use this "launch" approach if you have a simple app that creates a single WebView2 instance and that instance is active immediately upon launch.  In this scenario, there's one WebView2 instance, and it is available upon launch without needing to navigate through any native UI.  
+
+To drive WebView2 with Selenium and Microsoft Edge Driver:
 
-1.  To drive WebView2 with Selenium and Microsoft Edge Driver, first create the `EdgeOptions` object, by copying and pasting the following code snippet.
+1.  Create the `EdgeOptions` object, by copying and pasting the following code snippet:
 
     ```csharp
     static void Main(string[] args)
     {
         EdgeOptions edgeOptions = new EdgeOptions();
     ```
 
-1.  Next, we'll add code that does the following:
+    Next, we'll add code that does the following:
 
     *   Configure `edgeOptions` to use Chromium and WebView2, by setting the options `UseChromium` and `UseWebView` to `true`.
     *   Set `edgeOptions.BinaryLocation` to the file path of your host app binary.
```

Comment:
```suggestion
    *   Set `edgeOptions.BinaryLocation` to the file path of your WebView2 app binary.
```

Reply:
Suggesting "WebView2 app binary" here which should hopefully be more clear because it seems like we're no longer using the term "host app" in this document.